gitlab/v4/objects.py

```diff
@@ -2132,6 +2132,30 @@ def changes(self, **kwargs):
         path = '%s/%s/changes' % (self.manager.path, self.get_id())
         return self.manager.gitlab.http_get(path, **kwargs)
 
+    @cli.register_custom_action('ProjectMergeRequest', tuple(), ('sha'))
+    @exc.on_http_error(exc.GitlabMRApprovalError)
+    def approve(self,
+                sha,
```

Comment:
`sha` is optional so it would be best to have a default value.
```suggestion
                sha=None,
```

Could you also have all the parameters on one line (purely cosmetic)?

Reply:
Sure no problem.